### PR TITLE
Babylon balance changes update

### PR DESCRIPTION
**Pull request by Unintuition-Time**
Updates to the balance changes and finalizing tweaks to previous patches and merges.

@HiddeLekanne @Danarca

### CODE REVIEW

File: common/defines/babylon_defines.txt [hunk 1]
```diff
@@ -11,17 +11,17 @@
...
+	DEVASTATION_DECAY_RATE = -0.07 #-0.1 Per day, when there are no occupations
 	DEVASTATION_FROM_BATTLES = 0.15 # For each battle
 
 	AUTO_CAPITULATE_WAR_SUPPORT = -100 # When war support is this low, the country will auto capitulate
 	DAYS_BETWEEN_WAR_EXHAUSTION = 7 # Every this many days war support will be reduced by the war exhaustion formula
 	WAR_EXHAUSTION_BASE = 0.25
-	WAR_EXHAUSTION_KIA_FACTOR = 5.0 #20 # Scaled by casualties compared to total manpower
-	WAR_EXHAUSTION_KIA_BASE_MULTIPLIER = 0.05 #0.1	# War Exhaustion from casualties is always multiplied by at least this
+	WAR_EXHAUSTION_KIA_FACTOR = 2.0 #20 # Scaled by casualties compared to total manpower
+	WAR_EXHAUSTION_KIA_BASE_MULTIPLIER = 0.04 #0.1	# War Exhaustion from casualties is always multiplied by at least this
```
Comment (Danarca) [inline common/defines/babylon_defines.txt:21]: This is a big change, going from 5 to 2. Have you tested this?

File: common/modifiers/00_static_modifiers.txt [hunk 2]
```diff
@@ -366,7 +366,7 @@ political_dysfunction = {
 # Babylon RP: Changed value to 10% reduction (Danarca)
 diplomatic_mitigation = {
 	icon = gfx/interface/icons/timed_modifier_icons/modifier_flag_positive.dds
-	country_infamy_decay_mult = 0.10
+	country_infamy_decay_mult = 0.20
```
Comment (Danarca) [inline common/modifiers/00_static_modifiers.txt]: Another big change, what's the reasoning behind this? With the scaling infamy we solved a big problem already :s

Comment (Unintuition-Time) [reply common/modifiers/00_static_modifiers.txt]: This change was done pre-infamy modifiers, as far as I know, and now that we have actual penalties for Infamy we need proper decay that the game provides or else people are going to struggle to do what even historical nations did. I think 20 could be too high, but at least 15 is reasonable considering you have to actively not have any deals with other nations to keep that modifier at 100% effectiveness.

File: common/defines/babylon_defines.txt (hunk 1 above)
Comment (Unintuition-Time) [reply common/defines/babylon_defines.txt:21]: I tested it with 5 and it was still way too high, and with regards to casualties specifically I would rather it lower than higher importance-wise for war support. 2 is still huge, as at 100% casualties that'd still mean a nation at 100 war support would still drop out of the war in less than a year and a half if there is no wargoal set against them. The goal there is to get war length back to more rational levels, where full occupation and wargoal completion affect the support way more than kia count.

**Review by Danarca — APPROVED** [review]
Approved!

**Pull request merged**